ios/Capacitor/Capacitor/Plugins/CapacitorUrlRequest.swift

```diff
@@ -108,14 +108,23 @@ public class CapacitorUrlRequest: NSObject, URLSessionTaskDelegate {
         }
     }
 
+    @available(*, deprecated)
```

Comment:
can you add a message to the deprecation that says that the one with `[String: Any]` should be used instead?